croud/__main__.py

```diff
@@ -1206,6 +1210,91 @@
                         "resolver": org_files_delete,
                     },
                 }
+            },
+            "credits": {
+                "help": "Manage organization's credits.",
+                "commands" : {
+                    "list": {
+                        "help": "Lists all credits of an organization.",
+                        "extra_args": [
+                            Argument(
+                                "--org-id", type=str, required=True,
+                                help="The organization ID to use.",
+                            ),
+                            Argument(
+                                "--status", type=str, required=False,
+                                help="Filter credits by status. By default only "
+                                     "``ACTIVE`` credits are listed.",
+                            ),
+                        ],
+                        "resolver": org_credits_list,
+                    },
+                    "create": {
+                        "help": "Creates a new credit for the specified organization.",
+                        "extra_args": [
+                            Argument(
+                                "--org-id", type=str, required=True,
+                                help="The organization ID to use.",
+                            ),
+                            Argument(
+                                "--amount", type=float, required=True,
+                                help="The amount to be credited in USD.",
+                            ),
+                            Argument(
+                                "--expiration-date", type=str, required=True,
+                                help="The expiration date of the credit in ISO 8601 "
```

Comment:
Maybe also specify what that format is, cause most people will probably need to look it up ;))

```suggestion
                                help="The expiration date of the credit in ISO 8601 (i.e. 2024-01-01T10:00:00)"
```